src/main/java/net/hypixel/nerdbot/listener/ReactionChannelListener.java

```diff
@@ -0,0 +1,33 @@
+package net.hypixel.nerdbot.listener;
+
+import lombok.extern.log4j.Log4j2;
+import net.dv8tion.jda.api.entities.Message;
+import net.dv8tion.jda.api.events.message.MessageReceivedEvent;
+import net.dv8tion.jda.api.hooks.SubscribeEvent;
+import net.hypixel.nerdbot.NerdBotApp;
+import net.hypixel.nerdbot.channel.ReactionChannel;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+
+@Log4j2
+public class ReactionChannelListener {
+
+    private final List<ReactionChannel> reactionChannels = NerdBotApp.getBot().getConfig().getReactionChannels();
+
+    @SubscribeEvent
+    public void onMessageReceive(MessageReceivedEvent event) {
+        Optional<ReactionChannel> reactionChannel = reactionChannels.stream().filter(channel -> channel.getDiscordChannelId().equals(event.getChannel().getId())).findFirst();
+        if (reactionChannel.isPresent()) {
+            Message message = event.getMessage();
+            reactionChannel.get().getEmojiIds().stream()
+                    .map(emojiId -> NerdBotApp.getBot().getJDA().getEmojiById(emojiId))
+                    .filter(Objects::nonNull)
+                    .forEachOrdered(emoji -> {
+                        message.addReaction(emoji).queue();
+                        log.info("Added reaction '" + emoji.getName() + "' to message " + message.getId() + " in reaction channel " + reactionChannel.get().getName());
+                    });
+        }
```

Comment:
log some info if the reactionChannel cannot be found

Reply:
With the current logic it would trigger every time a user sends a message in _any_ channel in the guild. I don't think it would be worth

Reply:
hmmm okay all good then